misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobHandlerModule.kt

```diff
@@ -14,17 +17,21 @@ import kotlin.reflect.KClass
  */
 class AwsSqsJobHandlerModule<T : JobHandler> private constructor(
   private val queueName: QueueName,
-  private val handler: KClass<T>
+  private val handler: KClass<T>,
+  private val dependsOn: List<Key<out Service>> = listOf()
 ) : KAbstractModule() {
   override fun configure() {
     newMapBinder<QueueName, JobHandler>().addBinding(queueName).to(handler.java)
-    install(ServiceModule<AwsSqsJobHandlerSubscriptionService>()
-        .dependsOn<SqsJobConsumer>())
+    install(ServiceModule(AwsSqsJobHandlerSubscriptionService::class.toKey(),
+        dependsOn + listOf(SqsJobConsumer::class.toKey())))
```

Comment:
FYI @keeferrourke

Reply:
I’d love to give you a mechanism to do this elsewhere

Reply:
#1055